typescript/infra/scripts/utils.ts

```diff
@@ -20,6 +27,40 @@ export async function getCoreEnvironmentConfig(
   return (await import(moduleName(environment))).environment;
 }
 
+export async function getMultiProvider(
+  environment: DeployEnvironment,
+): Promise<MultiProvider<any>> {
+  return (await import(moduleName(environment))).getMultiProvider();
+}
+
+export async function getMultiProviderRemote<Networks extends ChainName>(
```

Comment:
I don't think we should overload `Remote` here

Reply:
Agreed, renamed to getMultiProviderFromGCP